blackbelt/dependencies.py

```diff
@@ -0,0 +1,353 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import re
+import os
+import json
+import shutil
+import tempfile
+import subprocess
+import itertools
+
+import click
+
+
+__all__ = ['check', 'parse_dep']
+
+
+# https://github.com/davglass/license-checker#custom-format
+LICENSE_CHECKER_FORMAT_KEYS = [
+    'name', 'repository', 'dependencyPath', 'url', 'version', 'licenses',
+    'licenseText', 'licenseModified', 'licenseFile', 'path',
+]
+
+# https://github.com/davglass/license-checker#custom-format
+LICENSE_CHECKER_FORMAT = {}
```

Comment:
Can be simplified using dict comprehension
```python
LICENSE_CHECKER_FORMAT = {i: None for i in LICENSE_CHECKER_FORMAT_KEYS}
```
Just uncovering my older python knowhow so pls bear with me ;]

Reply:
Yeah, right after dropping the Python 2.7 support 😅

Reply:
@honzajavorek dict comprehension is available in 2.* and also in 3.*

Reply:
You are right 🙀 Fixed!

Reply:
It feels like dejavu with PHP. I remember telling to ppl, that I am a PHP developer. And if somebody new what PHP was, they usually asked: "What version?"  ;]